tests/lib/outputs/csv/csv_test.py

```diff
@@ -0,0 +1,214 @@
+        assert (
+            output_data["REMEDIATION_RECOMMENDATION_URL"]
+            == "http://example.com/remediation"
+        )
+        assert output_data["REMEDIATION_CODE_NATIVEIAC"] == "native-iac-code"
+        assert output_data["REMEDIATION_CODE_TERRAFORM"] == "terraform-code"
+        assert output_data["REMEDIATION_CODE_CLI"] == "cli-code"
+        assert output_data["REMEDIATION_CODE_OTHER"] == "other-code"
+        assert output_data["COMPLIANCE"] == "compliance_key: compliance_value"
+        assert output_data["CATEGORIES"] == "category1,category2"
+        assert output_data["DEPENDS_ON"] == "dependency"
+        assert output_data["RELATED_TO"] == "related finding"
+        assert output_data["NOTES"] == "Notes about the finding"
+        assert output_data["PROWLER_VERSION"] == "1.0"
+
+    def test_csv_write_to_file(self):
+        mock_file = StringIO()
+        findings = [self.generate_finding()]
+
+        # Crear la instancia de CSV con mock_file como file descriptor
+        output = CSV(findings)
+        output._file_descriptor = (
+            mock_file  # Asignar StringIO como el descriptor de archivo
+        )
+
+        # Llamar a la función a testear
+        output.batch_write_findings_to_file()
+
+        # Volver al inicio del StringIO y leer el contenido
+        mock_file.seek(0)
+        content = mock_file.read()
+        assert "OAuth" in content
+        assert "12345" in content
+        assert "Example Account" in content
+        assert "example@example.com" in content
+        assert "org-123" in content
+        assert "Example Org" in content
+        assert "tag1 | tag2" in content
+        assert "finding-123" in content
+        assert "AWS" in content
+        assert "check-123" in content
+        assert "Example Check" in content
+        assert "Security" in content
+        assert "FAIL" in content
+        assert "Extended status" in content
+        assert "False" in content
+        assert "Example Service" in content
+        assert "Example Subservice" in content
+        assert "critical" in content
+        assert "Instance" in content
+        assert "resource-123" in content
+        assert "Example Resource" in content
+        assert "Detailed information about the resource" in content
+        assert "tag1,tag2" in content
+        assert "aws" in content
+        assert "us-west-1" in content
+        assert "Description of the finding" in content
+        assert "High" in content
+        assert "Recommendation text" in content
+        assert "native-iac-code" in content
+        assert "terraform-code" in content
+        assert "cli-code" in content
+        assert "other-code" in content
+        assert "compliance_key: compliance_value" in content
+        assert "category1,category2" in content
+        assert "dependency" in content
+        assert "related finding" in content
+        assert "Notes about the finding" in content
+        assert "1.0" in content
```

Comment:
It'd be ideal to verify that this information is correctly place, maybe you can read the whole line and assert it to:
```
AAA;BBB;CCC;DDD;EEE;
```

Reply:
Done